cylc/flow/cfgspec/suite.py

```diff
@@ -285,10 +284,13 @@
             Conf('abort on timeout', VDR.V_BOOLEAN)
             Conf('abort on inactivity', VDR.V_BOOLEAN)
             Conf('mail events', VDR.V_STRING_LIST, None)
-            Conf('mail from', VDR.V_STRING)
-            Conf('mail smtp', VDR.V_STRING)
-            Conf('mail to', VDR.V_STRING)
-            Conf('mail footer', VDR.V_STRING)
+
+        with Conf('mail'):
+            Conf('from', VDR.V_STRING)
+            Conf('smtp', VDR.V_STRING)
+            Conf('to', VDR.V_STRING)
+            Conf('footer', VDR.V_STRING)
+            Conf('task event batch interval', VDR.V_INTERVAL)
```

Comment:
It would be good to have a short description for each of these here? (At the moment there is a description for some of them later in the `[runtime]` section, but those ought to instead say that they override the setting here and link to the setting here)

Reply:
Of course. ATM I'm happy to let it slip as we need to have a complete run through of the config soon, just waiting for these conf changes to go in.

Reply:
Ok